sdk/web-pubsub/web-pubsub/src/normalizeOptions.ts

```diff
@@ -10,3 +10,11 @@ export default function normalizeSendToAllOptions<T extends { excludedConnection
     excluded: excludedConnections
   };
 }
+
+export function normalizeGenerateClientTokenOptions<T extends { roles?: string[] }>(options: T) {
+  const { roles, ...otherOptions } = options;
+  return {
+    ...otherOptions,
+    role: roles
+  };
```

Comment:
is all we're doing here renaming `roles` -> `role` ? If so, why not use a simple swagger transform to tweak the mapper?